### ISSUE TITLE
VS: Commit command is failing when the current tab is on a project properties

### ISSUE THREAD

**Issue by manuc66** [gitextensions/gitextensions#8258]
## Current behaviour

It is impossible to launch the commit window in Visual Studio when the active document is a project properties.

## Expected behaviour
The commit window should appear, allowing committing files

## Steps to reproduce

Right click on a .Net project, choose properties
Then click on the "Commit" button in the toolbar
An error popup is triggered with the following message : 
```
---------------------------
Microsoft Visual Studio
---------------------------
The parameter is incorrect. (Exception from HRESULT: 0x80070057 (E_INVALIDARG))
---------------------------
OK   
---------------------------
```

Other command affected:

- Browse
- Pull
- Push
- Stash
- View changes
- File history
- ...

## Screenshots

![image](https://user-images.githubusercontent.com/5827164/85234408-67f67100-b40d-11ea-9f43-181187013995.png)

## Did this work in previous version of GitExtensions

Multiple release are probably affected

## Environment

- GitExtensions version:
- GIT version: 2.27.0.windows.1
- OS version: Microsoft Windows [Version 10.0.18363.900]
- .NET version: .Net48

**Comment by vbjay** [comment]
This is because it can't get a path from it.

**Comment by manuc66** [comment]
@vbjay The extension https://github.com/heku/Kool.EditProject is able to edit the project of the current file and find its path. Maybe it might help to find a workaround.

**Comment by vbjay** [comment]
No. I mean the information from the editor of a code file is different than
the project properties dialog. The same thing happens on the overview page
when you start an asp .net project. It's the current active tab that gives
us the context to commit from what. Sometimes the active tab doesn't have
info we need.

**Comment by itsho** [comment]
I've already fixed this bug in this PR:
https://github.com/gitextensions/gitextensions/pull/7677

**Comment by pmiossec** [comment]
> I've already fixed this bug in this PR:
> #7677

OK. So it seems that a new vsix need to be released...
